Dockerfile

```diff
@@ -2,3 +2,9 @@ FROM unleashorg/unleash-server
 
 ENV HTTP_PORT 8080
 EXPOSE 8080
+
+COPY ./scripts/load_secrets_and_run.sh load_secrets_and_run.sh
+
+ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh", "./load_secrets_and_run.sh"]
+
+CMD ["node", "index.js"]
```

Comment:
After adding the ENTRYPOINT I tried starting the docker image but it wouldnt actually start the server so I think this is needed.